### ISSUE TITLE
Cannot get pride 0.11-rc-1 to work

### ISSUE THREAD

**Issue by mbezjak** [prezi/pride#154]
Consider the following gist: https://gist.github.com/mbezjak/f98c4aa52c767705f2d3 ([orig code](https://github.com/mbezjak/playground/blob/master/groovy/pride/generate-projects-and-run-pride.sh)).

When I run the script with pride version 0.10 (downloaded from [releases](https://github.com/prezi/pride/releases/download/0.10/pride-0.10.zip)), I get the desired output. First `gradle check` succeeds, the second one fails because of code changes (Library.getNumber() returns 11 instead of 42).

But the same example works differently in pride version 0.11-rc-1 (downloaded from [releases](https://github.com/prezi/pride/releases/download/0.11-rc-1/pride-0.11-rc-1.tar)). Now, both `gradle check` succeed. It's as if pride didn't convert dependency to `compile project(path: ":com.example:lib")` and thus doesn't see the change.

Even if I follow [updated README.md](https://github.com/prezi/pride/blob/gradle-dependency-substitution/README.md) (from PR [137](https://github.com/prezi/pride/pull/137)), and move `compile 'com.example:lib:0.1'` from *dynamicDependencies* to *dependencies*, it still behaves the same.

Any thoughts?

**Comment by lptr** [comment]
Which Gradle version are you using? Running Gradle with the generated wrapper (`./gradlew`) makes sure you are using the Gradle version Pride is designed to work with. In the case of 0.11 Pride will require Gradle 2.5+.

You also need to upgrade the Pride plugin in your projects to `0.11-rc-1` when using them with Pride 0.11-rc-1. In the future explicitly adding the the Pride plugin to your project is not going to be required. Now that Gradle 2.5 is out, I'm going to start working on these aspects.

You can check what's happening via `./gradlew app:dependencies --configuration compile`. This is what you should be seeing:

```text
:app:dependencies

------------------------------------------------------------
Project :app
------------------------------------------------------------

compile - Compile classpath for source set 'main'.
+--- com.example:lib:0.1 -> project :lib
|    \--- org.codehaus.groovy:groovy-all:2.4.4
\--- org.codehaus.groovy:groovy-all:2.4.4

(*) - dependencies omitted (listed previously)

BUILD SUCCESSFUL
```

The `com.example:lib:0.1 -> project :lib` is the important part.

**Comment by mbezjak** [comment]
Thanks for suggestions. `classpath "com.prezi.pride:gradle-pride-plugin:0.11-rc-1"` did the trick.

Didn't realize gradle plugin was linked to cmdline tool (README.md in PR [137](https://github.com/prezi/pride/pull/137) mentioned gradle plugin version 0.10).

Anyway, all seems fine now. `./gradlew app:dependencies --configuration compile` output is:

```
:app:dependencies

------------------------------------------------------------
Project :app
------------------------------------------------------------

compile - Compile classpath for source set 'main'.
+--- com.example:lib:0.1 -> project :lib
|    \--- org.codehaus.groovy:groovy-all:2.4.4
\--- org.codehaus.groovy:groovy-all:2.4.4

(*) - dependencies omitted (listed previously)

BUILD SUCCESSFUL
```

I guess this issue can be closed. There is nothing wrong with pride 0.11-rc-1.

**Comment by lptr** [comment]
Great. I just released [0.11](https://github.com/prezi/pride/releases/tag/0.11).
